src/sentry/api/helpers/group_index/delete.py

```diff
@@ -120,6 +121,9 @@ def delete_groups(
     if not group_list:
         return Response(status=204)
 
+    if any([group.issue_category == GroupCategory.PERFORMANCE for group in group_list]):
+        return Response({"detail": "Cannot delete performance issues."}, status=403)
```

Comment:
Not a blocker, but it would be nice to start using rest framework exceptions instead
```suggestion
        raise rest_framework.exceptions.PermissionDenied(detail="Cannot delete performance issues.")
```